webapi/core.py

```diff
@@ -92,7 +92,7 @@ def render_patch(self, request):
             request.setHeader('Access-Control-Allow-Methods', 'POST')
             request.setHeader('Access-Control-Allow-Credentials', 'true')
 
-        request.write('')
+        request.write(b'')
```

Comment:
I don't know for certain that this is backwards compatible

Reply:
It could be done in the following manner:
1. check what deluge version is available
2. for versions wing bytes use bytes, for lower versions use str as before

Reply:
I don't actually know in which deluge version this breaking change occurred, I will just assume that it was 2.1.0

Reply:
Yes, presumably so, since 2.1 is the version in that py2 support was dropped
https://deluge.readthedocs.io/en/latest/changelog.html#breaking-changes

Reply:
done